helix-term/src/commands/typed.rs

```diff
@@ -2454,6 +2455,37 @@ fn yank_diagnostic(
     Ok(())
 }
 
+fn read_file_info_buffer(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+
+    let (view, doc) = current!(cx.editor);
+
+    ensure!(!args.is_empty(), "file name is expected");
+
+    let filename = args.get(0).unwrap();
+    let path = PathBuf::from(filename.to_string());
+    if !path.exists() {
+        bail!("file doesn't exist: {}", filename);
+    }
```

Comment:
Here we should also check whether the file is not a directory:

```suggestion
    ensure!(
        path.exists() && path.is_file(),
        "path is not a file: {:?}",
        path
    );
```